Further integrate SDW docs, initial work for Installation and Migration documentation, eliminate need to jump around docs (#782)
freedomofpress/securedrop-docs: docs/admin/installation/intro_for_admins.rst

For larger newsrooms, there may be a team of systems admins, but at least one person within the organization will need to serve as the administrator. In some situations, such as smaller news organizations where a journalist has the technical capacity to administer systems, one person can serve as both Journalist and Administrator. When possible, we advise having a dedicated staff member serving the role of SecureDrop Administrator.

The admin connects to the *Application* and *Monitor Servers* over `authenticated onion services <https://tb-manual.torproject.org/onion-services/>`__, and manages them using `Ansible <https://www.ansible.com/>`__.

If you are considering becoming a SecureDrop administrator, below are some
attributes that will be important to have:

* Experience with managing Linux-based systems from the command line.
* Proficiency with network hardware such as firewalls and switches (e.g. pfSense).
* Experience with QubesOS.
* Experience with configuration management tools such as Ansible, Salt, Chef, or Puppet.
* Ability to use and configure secure communication tools such as GPG.

We consider the first two requirements and the last three preferred attributes.

> This Admin Guide will walk you through the entire experience, from planning to installation, to deployment, to ongoing maintenance.

Reviewer: ```suggestion
This Admin Guide covers planning, installation, deployment, and ongoing maintenance.
```
